tests/utils/test_uri.py

```diff
@@ -355,7 +356,8 @@ def test_add_databricks_profile_info_to_artifact_uri(artifact_uri, profile_uri,
 
 @pytest.mark.parametrize("artifact_uri, profile_uri", [
     ('dbfs:/path/a/b', 'databricks://not:legit:auth'),
-    ('dbfs:/path/a/b/', 'databricks://scope/key/'),
```

Comment:
^In this scenario, we will just interpret the "scope" component and ignore the rest, so it no longer throws.